json-body-transformer/src/main/java/com/ninecookies/wiremock/extensions/JsonBodyTransformer.java

```diff
@@ -31,8 +31,10 @@
 
 	private static final String CONTENT_TYPE_APPLICATION_JSON = "application/json";
 
+	private final Random random = new Random(System.currentTimeMillis());
```

Comment:
Hm, maybe that is a c# relict in my head (as msdn recommends to specify a seed), but as I know figured out `Random()` constructor does `this(seedUniquifier() ^ System.nanoTime());` which should be in fact random enough... :)